src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -252,6 +300,14 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
                             recurrencePeriod = 0;
                             listener().getLogger().println("After 10s process did not stop");
                             getContext().onFailure(cause);
+                            try {
+                                FilePath workspace = getWorkspace();
+                                if (workspace != null) {
+                                    controller.cleanup(workspace);
+                                }
+                            } catch (IOException | InterruptedException x) {
+                                Functions.printStackTrace(x, listener().getLogger());
```

Comment:
🐜 Might be worth mentioning that we failed to do workspace cleanup (so it's clear this is not super serious).

Reply:
Could do that.

Reply:
Cancel that—at this point we have already printed an unusual message above, and any exception here probably came from `getWorkspace`.